cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -542,8 +549,9 @@ policy "aws_ri_recommendations" do
       field "licenseModel" do
         label "License Model"
       end
-      field "lookbackPeriodInDays" do
+      field "lookBackPeriod" do
```

Comment:
should the B in Back be capitalized?

Reply:
Yes. According to shared new documentation for normalization, they capitalize "B" for that field.